bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/HCaptcha.java

```diff
@@ -0,0 +1,27 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2024 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.forms.core.components.models.form;
+
+import org.osgi.annotation.versioning.ConsumerType;
+
+/**
+ * Defines the form {@code HCaptcha} Sling Model used for the {@code /apps/core/fd/components/form/hcaptcha/v1/hcaptcha}
+ * component.
+ *
+ * @since com.adobe.cq.forms.core.components.models.form 5.4.0
+ */
+@ConsumerType
+public interface HCaptcha extends Captcha {}
```

Comment:
we should not need this interface if its not adding any new method over the base interface.

Reply:
This is just a marker interface to be consumed by HCaptchaImpl.

Reply:
Why can't the impl just implement Captcha?

Reply:
This is consumed by sightly, if we use HCaptchaImpl implemented Captcha then it would have not picked up the correct impl.